db/migrate/20190612021005_rename_requester_to_owner.rb

```diff
@@ -0,0 +1,5 @@
+class RenameRequesterToOwner < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :requests, :requester, :owner
```

Comment:
This migrate has problem to keep old data valid. The old requester column stored First and Last name while the new owner column stores username. I don't have better solution. Maybe it is OK to break backward compatibility.